JWBestPracticeApps/DRM Fairplay/DRM Fairplay/ViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ViewController.swift
+//  DRM Fairplay
+//
+//  Created by David Almaguer on 06/08/21.
+//
+
+import UIKit
+import JWPlayerKit
+
+private let EZDRMLicenseAPIEndpoint = "https://fps.ezdrm.com/api/licenses"
+private let EZDRMCertificateEndpoint = "https://fps.ezdrm.com/demo/video/eleisure.cer"
+private let EZDRMVideoEndpoint = "https://fps.ezdrm.com/demo/video/ezdrm.m3u8"
+
+class ViewController: JWPlayerViewController,
+                      JWDRMContentKeyDataSource {
+
+    var contentUUID: String?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Set up the player.
+        setUpPlayer()
+    }
+
+    /**
+     Sets up the player with a DRM configuration.
+     */
+    private func setUpPlayer() {
+        let videoUrl = URL(string:EZDRMVideoEndpoint)!
+
+        // First, use the JWPlayerItemBuilder to create a JWPlayerItem that will be used by the player configuration.
+        let playerItembuilder = JWPlayerItemBuilder()
+            .file(videoUrl)
+
+        var playerItem: JWPlayerItem!
+        do {
+            playerItem = try playerItembuilder.build()
+        } catch {
+            // Handle player item build failure
+            print(error.localizedDescription)
+            return
+        }
+
+        // Second, create a player config with the created JWPlayerItem.
+        let configBuilder = JWPlayerConfigurationBuilder()
+            .playlist([playerItem])
+            .autostart(true)
+        var config: JWPlayerConfiguration!
+        do {
+            config = try configBuilder.build()
+        } catch {
+            // Handle player item build failure
+            print(error.localizedDescription)
+            return
+        }
```

Comment:
Instead of multiple do-catch, why not wrap everything in a do-catch? it'll make the code cleaner and easier to understand.

Reply:
Addressed